src/components/OSCALControlModification.js

```diff
@@ -138,8 +137,9 @@ export default function OSCALControlModification(props) {
     modLength += len;
   }
 
-  // TODO(EGRC-407): Implement remove modifications
-  // if (alter.removes) { }
+  // TODO: There should also be a way to get all remove modifications so that
+  // users can better understand control modifications with an OSCAL Profile.
+  // https://github.com/EasyDynamics/oscal-react-library/issues/89
```

Comment:
This one can be removed entirely. There's no supporting code. It's just an entirely new feature that is tracked in the referenced issue.